ansible/roles/vm_set/library/kickstart.py

```diff
@@ -211,7 +211,7 @@ def main():
         result = {'kickstart_code': -4, 'changed': False, 'msg': "Match for output isn't found"}
     except ENotInEnabled:
         result = {'kickstart_code': -5, 'changed': False, 'msg': "Not in enabled mode"}
-    except Exception, e:
+    except Exception as e:
```

Comment:
is this the only issue to support py3?

can you check this pr #1771

Reply:
yes, it's only py3 issue in that script preventing that scenario to be ran.